### PR TITLE
completes Q10 GET all users

### PR DESCRIPTION
**Pull request by robbochobbo**
added GET /api/users endpoint

### CODE REVIEW

**Comment by robbochobbo** [comment]
Whoops Q10 not Q9

File: __tests__/app.test.js [hunk 1]
```diff
@@ -333,3 +333,29 @@ describe('/api/comments/comment_id', () => {
...
+            expect(users.length).toBe(4)
+            users.forEach((user) => {
+                expect(typeof user.username).toBe('string')
+                expect(typeof user.name).toBe('string')
+                expect(typeof user.avatar_url).toBe('string')
+            })
+        })
+    })
+
+    test('GET 200: responds with users sorted by username, alphabetically', () => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js:351]: nice thinking

File: models/models.js [hunk 2]
```diff
@@ -127,12 +127,22 @@ const removeCommentById = (comment_id) => {
     })
 }
 
+const fetchAllUsers = () => {
+    return db
+    .query(`SELECT * FROM users
```
Comment (roisinoreilly) [inline models/models.js:132]: really nice

**Review by roisinoreilly — COMMENTED** [review]
nice work on this ticket and I really like that you though to sort the users! great job :)

**Pull request merged**